Python/heap.py

```diff
@@ -0,0 +1,71 @@
+# Implementation of heap from list in Python
+
+class Heap(object):
+
+    def __init__(self):
+        self.list = []
+        self.last_index = -1
+
+    def insert(self, x):
+        self.last_index += 1
+        if not self.list:
+            self.list.append(x)
+            return
+        self.list.append(x)
+        index = self.last_index
+        parent = int((index-1)/2)
+        while (self.list[index] > self.list[parent]) and parent >= 0:
+            temp = self.list[index]
+            self.list[index] = self.list[parent]
+            self.list[parent] = temp
+            index = parent
+            parent = int((parent-1)/2)
+
+    def max(self):
+        return self.list[0]
+
+    def pop(self):
```

Comment:
This needs a comment for sure. Should not have to read the code to learn that pop returns the max element of the heap.